.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
```

Comment:
does anyone _not_ have these settings by default in their editor? and does having a linter ( `goodparts` ) not enforce them in the project? (_just curious ..._ 😉 )